egs/grid/s5/cmd.sh

```diff
@@ -22,8 +22,18 @@ export mkgraph_cmd="queue.pl -l arch=*64,ram_free=4G,mem_free=4G"
 #export cuda_cmd="queue.pl -q long.q@pcspeech-gpu"
 #export mkgraph_cmd="queue.pl -q all.q@@servers -l ram_free=4G,mem_free=4G"
 
-#c) run it locally...
-#export train_cmd=run.pl
-#export decode_cmd=run.pl
-#export cuda_cmd=run.pl
-#export mkgraph_cmd=run.pl
+
+#c) USFD cluster options
+config="conf/queue_usfd.conf"
+#export train_cmd="queue.pl  --config $config --mem 4G --rmem 4G"
+#export decode_cmd="queue.pl  --config $config --mem 4G --rmem 4G"
+#export mkgraph_cmd="queue.pl  --config $config --mem 4G --rmem 4G"
+#export cuda_cmd="queue.pl  --config $config --mem 24G --rmem 20G --gpu 1 --time 24:00:00"
+
+
+#d) run it locally...
+export train_cmd=run.pl
```

Comment:
I think the default config should be
export train_cmd="queue.pl --mem 2G"
export decode_cmd="queue.pl --mem 2G"
export mkgraph_cmd="queue.pl --mem 4G"
export cuda_cmd="queue.pl --gpu 1"

not the run.pl commands